### PR TITLE
Removed requirement for `design` column from experiments endpoint

### LINKED ISSUE #32: Remove requirement for `design` column from experiments endpoint
Currently the `design` column is required in the CSV file. This requirement should be removed from the code

### PR DESCRIPTION
**Pull request by im-prakher**
Fixed #32

### CODE REVIEW

File: app/yaba.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ paths:
                 CSV:
                   summary: A CSV file for experiment metadata Upload.Below is the schema of csv file with example row.
                   text/csv: |-
-                    name,start_date,end_date,description,design
+                    name,start_date,end_date,description
                     'MAC Season 8: Border','2019-01-01','2019-03-31','ProMix BRK20 14-14-14 Osmocote pots; pre-filled by Hummert Sorghum seed Conviron Growth House'....,'Grain Sorghum also planted to fill out empty plots'
```
Comment (rahul799) [inline app/yaba.yaml]:
```suggestion
                    'MAC Season 8: Border','2019-01-01','2019-03-31','ProMix BRK20 14-14-14 Osmocote pots; pre-filled by Hummert Sorghum seed Conviron Growth House'
```
I guess this must be removed too.

**Review by rahul799 — CHANGES_REQUESTED** [review]

**Review by Chris-Schnaufer — APPROVED** [review]

**Comment by KristinaRiemer** [comment]
I'm getting this error when I run the curl statement for the experiments endpoint: 

```
{
  "Message": "(psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint \"unique_name_per_species\""
}
```
Some more info about the error: 
```
yaba_api_1     | psycopg2.errors.NotNullViolation: null value in column "description" violates not-null constraint
yaba_api_1     | DETAIL:  Failing row contains (99000000025, MAC Season 8: Border , 2019-01-01, 2019-03-31, null, , 99000000036, 2020-03-05 17:50:18.732026, 2020-03-05 17:50:18.732026).
```

File: input_files/experiments.csv [hunk 2]
```diff
@@ -1,6 +1,6 @@
-name,start_date,end_date,description,design
-MAC Season 8: Border ,2019-01-01,2019-03-31,wefu,
-MAC Season 8: Uniformity ,2019-01-01,2019-03-31,some text,
-MAC Season 8: DP4 ,2019-01-01,2019-03-31,wefgyu,
-MAC Season 8: DP ,2019-01-01,2019-03-31,some text,
-MAC Season 8: DP test,2019-01-01,2019-03-31,some text,
```
Comment (rahul799) [inline input_files/experiments.csv:6]: The error is happening due to the deletion in these lines. The description row contains the value wefu, some text, etc.

**Review by rahul799 — CHANGES_REQUESTED** [review]

**Comment by im-prakher** [comment]
Sorry for catching up so late, got involved in some health issues but now I am fine. I think this will work now

**Comment by dlebauer** [comment]
@im-prakher could you please confirm that this makes the design column optional rather than removing it?

**Comment by im-prakher** [comment]
Yeah! I will look into this

**Comment by im-prakher** [comment]
I have finally removed the requirement for the design column, earlier there was an issue with CSV format of experiments.csv now I have fixed it.

**Pull request merged**